examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/container/.content.xml

```diff
@@ -4,4 +4,4 @@
     jcr:title="Adaptive Form Container"
     jcr:description="Add an Adaptive Form to a page."
     sling:resourceSuperType="core/fd/components/form/container/v2/container"
-    componentGroup=".hidden"/>
+    componentGroup="Core Components Examples - Adaptive Form"/>
```

Comment:
it should not be exposed for now till we finish with forms+sites development.
whenever we will expose it, we should be adding the sites component grp in future.

Reply:
Will raise another PR with just dor related bugs and skip enabling these for now